src/unix/qnx.c

```diff
@@ -0,0 +1,93 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
Please use this header: https://github.com/libuv/libuv/blob/2967557b6651ee10dd2f9e0b4baeff35df0ab684/src/unix/random-getrandom.c#L1-L20